### PR TITLE
sprint-automation: digest issues ready for acceptance

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Currently looks like:

Screenshot from 2021-07-06 12-13-01

SVG based images for avatars hosted on Jira are not possible to display in Slack so we can't see what is a sub-task/bug/etc and users with icons anywhere but gravatar don't display, but /shrug

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
xref https://dptp-robot-testing.slack.com/archives/C027534HBPF/p1625598728006100

File: cmd/sprint-automation/main.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+						},
+						&slack.TextBlockObject{
+							Type: slack.MarkdownType,
+							Text: issue.Fields.Assignee.DisplayName,
+						},
+					},
+				},
+			}
+		}
+		// we really don't want these things to line wrap, so truncate the summary
```
Comment (droslean) [inline cmd/sprint-automation/main.go:140]: Why? Does slack has any limit or we don't want to create big messages?

**Review by droslean — APPROVED** [review]

File: cmd/sprint-automation/main.go (hunk 1 above)
Comment (stevekuznetsov) [reply cmd/sprint-automation/main.go:140]: Slack has a limit to the width of an app's post, so if you use the full title it just line-wraps and makes the thing hard to read

**Pull request merged**